devel/checkmake/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/mrtazz/checkmake 0.2.2
```

Can you add `revision`?

```suggestion
go.setup            github.com/mrtazz/checkmake 0.2.2
revision            0
```